README.md

```diff
@@ -32,6 +32,22 @@ following to install them or update when pulling the utk-web repository:
 
 Then open [http://localhost:3000](http://localhost:3000) in a web browser.
 
+#### Running utk-web in Docker
+
+If want to avoid npm here is how to use Docker container. Build Docker image:
```

Comment:
Since `npm` would still be run in Docker, let's maybe say something like "To avoid running `npm` on your host directly, you can build a Docker image:". WDYT?

Reply:
Sure, sounds correct.